configs/det/dbnet/README.md

```diff
@@ -56,9 +56,26 @@ combination of these two modules leads to scale-robust feature fusion.
 DBNet++ performs better in detecting text instances of diverse scales, especially for large-scale text instances where
 DBNet may generate inaccurate or discrete bounding boxes.
 
-## 2. Results
 
-DBNet and DBNet++ were trained on the ICDAR2015, MSRA-TD500, SCUT-CTW1500, Total-Text, and MLT2017 datasets. In addition, we conducted pre-training on the SynthText dataset and provided a url to download pretrained weights. All training results are as follows:
+## 2. General purpose models
+
+Here we present general purpose models that were trained on wide variety of tasks (real-world photos, street views, documents, etc.) and challenges (straight texts, curved texts, long text lines, etc.) with two primary languages: Chinese and English. These models can be used right off-the-shelf in your applications or for initialization of your models.
+
+The models were trained on 12 public datasets (CTW, LSVT, RCTW-17, TextOCR, etc.) that contain wide range of images. The training set has 153,511 images and the validation set has 9,786 images.<br/>
+The test set consists of 598 images manually selected from the above-mentioned datasets.
+
+<div align="center">
+
+| **Model** | **Context**    | **Backbone** | **Languages**     | **F-score on Our Test Set** | **Throughput** | **Download**                                                                                             |
+|-----------|----------------|--------------|-------------------|:---------------------------:|----------------|----------------------------------------------------------------------------------------------------------|
+| DBNet     | D910x8-MS2.0-G | ResNet-50    | Chinese + English |           83.41%            | 256 img/s      | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_ch_en_general-a5dbb141.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_ch_en_general-a5dbb141-912f0a90.mindir) |
+| DBNet++   | D910x4-MS2.0-G | ResNet-50    | Chinese + English |           84.30%            | 256 img/s      | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50_ch_en_general-884ba5b9.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50_ch_en_general-884ba5b9-b3f52398.mindir) |
+</div>
```

Comment:
Please specify the data shape when exporting mindir below this table, like the ICDAR2015 one.